charts/mlp/tests/mlp_cm_tests.yaml

```diff
@@ -0,0 +1,71 @@
+suite: Unit tests for mlp config map values
+templates:
+  - configmap.yaml
+tests:
+  - it: should set the configs from values if no global values found
+    set:
+      global: {}
+      deployment.apiHost: "http://mlp/v1"
+      deployment.oauthClientID: "oauthId"
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - matchRegex:
+          path: metadata.name
+          pattern: mlp-*-config$
+      - matchRegex:
+          path: data.[mlp-config.yaml]
+          pattern: |
+            (?s).*apiHost: "http://mlp/v1"
+            .*oauthClientID: "oauthId"
+            .*applications:
+            .*api: \/api\/merlin\/v1
+            .*homepage: \/merlin
+            .*name: Merlin
+            .*api: \/api\/turing\/v1
+            .*homepage: \/turing
+            .*name: Turing
+            .*api: \/feast\/api
+            .*homepage: \/feast
+            .*name: Feast
+  - it: should set the configs from global if global values found
+    set:
+      global:
+        oauthClientID: "test-client-123"
+        mlp:
+          apiPrefix: "/v1"
+          vsPrefix: "/api"
+          uiPrefix: "/"
+        merlin:
+          apiPrefix: "/v1"
+          vsPrefix: "/api/merlin"
+          uiPrefix: "/merlin"
+        turing:
+          apiPrefix: "/v1"
+          vsPrefix: "/api/turing"
+          uiPrefix: "/turing"
+        feast:
+          apiPrefix: "/api"
+          vsPrefix: "/feast"
+          uiPrefix: "/feast"
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - matchRegex:
+          path: metadata.name
+          pattern: mlp-*-config$
+      - matchRegex:
+          path: data.[mlp-config.yaml]
+          pattern: |
```

Comment:
is there any support for yq/jq-format queries?

Reply:
yes. check https://github.com/quintush/helm-unittest/blob/master/DOCUMENT.md#assertion

Map keys in path containing periods (.) are supported with the use of a jq-like syntax:

```
- equal:
    path: metadata.annotations.[kubernetes.io/ingress.class]
    value: nginx
```

Reply:
oh, nice! for this configmap, it might not be the best way of testing, however, for other manifests, it should look much better

Reply:
yes. Because this is a big string under one key, I have no other option than use regex with the `s` flag to ignore the newlines.